shared/fileStorage.ts

```diff
@@ -116,7 +123,11 @@ export async function writeToBackupFile(data): Promise<string> {
   const fileName = getBackupFileName();
   const isDirectoryExists = await exists(backupDirectoryPath);
   if (isDirectoryExists) {
-    await removeFile(backupDirectoryPath);
+    //remove old backup
```

Comment:
are we doing this already?

Reply:
The comment I mean

Reply:
yes we are doing removing old backup in upcoming lines

Reply:
DO we need that comment then ?

Reply:
Removes the comment